examples/resourceServer/client/index.html

```diff
@@ -1,26 +1,51 @@
 <head>
-    <title>resourceServer</title>
+    <title>ResourceServer</title>
 </head>
 
 <body>
-    <h1>Welcome to resourceServer!</h1>
+    <h1>Welcome to ResourceServer!</h1>
 
     {{> authorize}}
 </body>
 
 <template name="authorize">
+    <p>
+      This <code>meteor-oauth2-server</code> package currently provides functionality for both the AuthenticationServer and the ResourceServer.  This functionality may be split into separate packages and/or examples in the future; however, for the time being, AuthenticationServer and ResourceServer are the same application.
+    </p>
     <p>The following steps represent the steps that are necessary to properly negotiate an oauth2 token. While these
-    steps work on this site, this site is for testing and demo purposes only. Do not use this configuration in production.</p>
-    <p>You'll know if everything is working if you can get to Step 6 and the user ids match.</p>
+    steps work on this site, this site is for testing and demo purposes only. Do not use this configuration in production.  You'll know if everything is working if you can get to Step 6 and the user ids match.</p>
     <hr />
 
-    <h2>Step 0 - Add an an authorized client</h2>
+    <h2>Step C1.1 - Generate Client ID & Secret</h2>
```

Comment:
While I agree with your suggestion would work in a production environment, we are working with examples that are intended to give a concise view of how to use the package. The idea of using the form and pre-porulating with defaults seems best. It also matches the UX from the clientApplication too.

I see an argument for keeping the code in place on Meteor.startup as guidance to the dev. At the same time, it's expected that the dev would understand howto automate this. Unless I'm mistaken, your suggestions indicate a motivation to use the example as a template project. Is this true?

Ultimately, it seems like the code in Meteor.startup would be useless and just clutter the example.